packages/util/src/logger/cloud-sdk-logger.ts

```diff
@@ -4,9 +4,13 @@ import {
   LoggerOptions as WinstonLoggerOptions,
   transports
 } from 'winston';
+import cds from '@sap/cds';
 import { kibana, local } from './format';
 
-const format = process.env.VCAP_SERVICES ? kibana : local;
+const cdsKibana = (cds.env as any).features?.kibana_formatter;
```

Comment:
[req] We should not do this in the SDK. Instead we need to provide some API, where users can set the format to `kibana`. It is then up to CAP to set it, when someone enables the Kibana format through `cds.env.features.kibana_formatter = true`.

(The default check for NODE_ENV is perfect though.)